process/README.md

```diff
@@ -44,7 +44,7 @@ Run the [Agent's status subcommand][5] and look for `process` under the Checks s
 The following metrics are not available on Linux or macOS:
 - Process I/O metrics are **not** available on Linux or macOS since the files that the Agent reads (`/proc//io`) are only readable by the process's owner. For more information, [read the Agent FAQ][6].
 
-The following metrics are not available on Windows:
+The following metrics are not available on [Windows][12]:
```

Comment:
I'm not sure it makes sense to put the ddagentuser link here. could it be down below in the new section instead?